frontend/assets/stylesheets/components/_articles.scss

```diff
@@ -244,12 +244,19 @@
     float: left;
     padding-right: $gs-gutter/3;
 
-    @include mq(tablet) {
+    @include mq(wide) {
         @include fs-header(4);
         line-height: get-line-height(header, 2);
```

Comment:
I'm not sure about the font-only mix-in, but when trying to do mq queries without the repetition, the precedence rules were confusing me and I couldn't get a consistent result.

If you've got time to show me, I'd be happy to try though.